front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -80,6 +81,7 @@ const usedModelConfigs = [
   CLAUDE_DEFAULT_MODEL_CONFIG,
   CLAUDE_INSTANT_DEFAULT_MODEL_CONFIG,
   MISTRAL_7B_DEFAULT_MODEL_CONFIG,
+  MISTRAL_SMALL_MODEL_CONFIG,
```

Comment:
This is unrelated to the addition of mistral to core + dev tools let's keep it for the next PR :+1: